buildlib/io_demo/az-stage-io-demo.yaml

```diff
@@ -14,12 +16,15 @@ steps:
       downtime=$(downtime) \
       uptime=$(uptime) \
       |& add_timestamp &>corrupter.log &
-    pgrep -u "$USER" -f 'network-corrupter'
+    while ! pgrep -u "$USER" -f 'network-corrupter'
+    do
+      sleep 1
+    done
     corrupter_pid=$(pgrep -u "$USER" -f 'network-corrupter')
     echo "corrupter_pid=$corrupter_pid"
     azure_set_variable "corrupter_pid" "$corrupter_pid"
   displayName: Start network corrupter
-  timeoutInMinutes: 15
+  timeoutInMinutes: 3
```

Comment:
should be separate PR that we will merge to fix build
the rest - will not be merged since it will fail the CI